task/net/src/bsp_support.rs

```diff
@@ -0,0 +1,86 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Common bits for Board Support Packages (BSPs).
+//!
+//! This would be named `bsp` but the BSP infrastructure squats on that name.
+//!
+//! # How the BSP stuff works
+//!
+//! The netstack defines a `bsp` module, but chooses which source file to use
+//! based on the board ID/rev, selecting among options in `src/bsp/`.
+//!
+//! A BSP module is expected to export a single type, called `BspImpl`, which
+//! implements the `Bsp` trait from this module.
+
+use drv_stm32h7_eth as eth;
+// This is not a typo: we use the vsc7448 type on all BSPs.
```

Comment:
This is probably meant to be a few lines down

Reply:
Ah yeah; I think that you could get away with it if you included a whitespace line separating this import from the others, but deleting it is fine by me.